app/ante/evm/09_gas_consume.go

```diff
@@ -79,18 +79,6 @@ func deductFees(
 		return nil
 	}
 
-	// If the account balance is not sufficient, try to withdraw enough staking rewards
-	if err := anteutils.ClaimStakingRewardsIfNecessary(
-		ctx,
-		keepers.Bank,
-		keepers.Distribution,
-		keepers.Staking,
-		feePayer,
-		fees,
-	); err != nil {
-		return err
-	}
-
```

Comment:
I think this was merged on `main` already no? Wondering why it comes up as a change here and if it remains after merging

Reply:
The target branch here is the v20 release branch. We are trying to separate the ICS stuff from what's going on main right now.

Reply:
ohhh gotcha sorry - I think you can also remove the rest of the stuff as well then as on #2719